apply-css/background.js

```diff
@@ -29,9 +29,8 @@ function toggleCSS(tab) {
 Returns true only if the URL's protocol is in APPLICABLE_PROTOCOLS.
 */
 function protocolIsApplicable(url) {
-  var anchor =  document.createElement('a');
-  anchor.href = url;
-  return APPLICABLE_PROTOCOLS.includes(anchor.protocol);
+  const protocol = (new URL(url)).protocol;
```

Comment:
Add comment that `url` must be a valid URL, or else the `URL` constructor throws.

Reply:
That is an interesting distinction I did not realize.  I've added it to the doc string of the function.  Is this guaranteed to be the case for URLs in the tab bar?  I interpreted [MDN Tab#Type](https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/API/tabs/Tab#Type) to indicate this is safe.

Reply:
Since the `tabs` permission is present, and `tab.url` is an absolute URL, this is indeed the case.
Without the `tabs` permission, `tab.url` could have been `undefined`, and cause the `URL` constructor to throw.